courses-app/src/App.jsx

```diff
@@ -1,7 +1,27 @@
+import { useState } from 'react';
+import Header from './components/Header/Header';
+import Courses from './components/Courses/Courses';
+import CreateCourse from './components/CreateCourse/CreateCourse';
+
 import './App.css';
 
 function App() {
-	return <div className='App'>START</div>;
+	const [isCreateCourse, setIsCreateCourse] = useState(false);
+
+	function switchCourseCreateCourse() {
```

Comment:
Also naming.
Here we have only 2 states: change route to "Courses List" or to "Add Course"
I think _toggleRoute_ represents better the job of this function